src/Document.php

```diff
@@ -689,6 +694,16 @@ public function canViewFile(array $options = [])
             return true;
         }
 
+        if (count($options)) {
```

Comment:
I guess it would be easier to handle directly `$options["itemtype"]` and `$options["items_id"]` here.